test/RiskManagerV1.test.js

```diff
@@ -43,9 +46,9 @@ describe("RiskManagerV1", () => {
     masterAuction = await Auction.deploy()
     await masterAuction.deployed()
 
-    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1")
+    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1Stub")
```

Comment:
Nitpick: can we `s/RiskMangerV1/RiskManagerV1Stub`? We use this pattern in other test files where we have to work with stubs.

Reply:
https://github.com/keep-network/coverage-pools/pull/43/commits/0fd0f595c291327882884f51b896154a28b23fac